stable_diffusion_ov/pipelines/unet_loop_split.cpp

```diff
@@ -18,21 +18,8 @@ namespace cpp_stable_diffusion_ov
         ov::Core& core)
         : _device_unet_positive_prompt(device_unet_positive_prompt), _device_unet_negative_prompt(device_unet_negative_prompt)
     {
-        //Read the OpenVINO encoder IR (.xml/.bin) from disk, producing an ov::Model object.
-        auto model = core.read_model(model_path);
-
-        logBasicModelInfo(model);
-
         _sample_shape = ov::Shape{ 1, 4, img_height / 8, img_width / 8 };
```

Comment:
This shape works even for batch 2? I would have expected that first dimension to be batch size.

Reply:
Yes it works, although it's kind of confusing because the shape that we are setting here is only used to check the size of the ```latents``` tensor that is passed into the ```operator()``` function (here: https://github.com/intel/stablediffusion-pipelines-cpp/blob/sd1.5_batch2/stable_diffusion_ov/pipelines/unet_loop_split.cpp#L55). Regardless of whether unet is batch1 or batch2, *latents* should have a batch size of 1.